locales/en/messages.json

```diff
@@ -1981,7 +1981,7 @@
     },
 
     "receiverHelp": {
-        "message": "Please read receiver chapter of the documentation.  Configure serial port (if required), receiver mode (serial/ppm/pwm), provider (for serial receivers), bind receiver, set channel map, configure channel endpoints/range on TX so that all channels go from ~1000 to ~2000.  Set midpoint (default 1500), trim channels to 1500, configure stick deadband, verify behaviour when TX is off or out of range.<br /><span class=\"message-negative\">IMPORTANT:</span> Before flying read failsafe chapter of documentation and configure failsafe."
+        "message": "<b>Basic setup</b>: For serial connections, go to the Ports tab and enable ‘Serial Rx’ for the UART to which the Rx is connected.  In the Receivers tab, choose ‘Serial based receiver…’, and set the correct link/data format at the ‘Serial Rx Provider’ drop-down.  For other types of receiver, set the correct mode (ppm, pwm etc).  Choose the correct ‘Channel Map’ for your radio.  Bind the receiver and check that the Roll, Pitch and other channel bar graphs move correctly when you move the sticks.  You may need to reconfigure Map, Mode, Channels or Mix so that, for example, moving the Roll stick changes Roll values.  Adjust the channel endpoint or range values in the transmitter so that all channels go from ~1000 to ~2000.  Trim the midpoint to 1500.  RC Smoothing is best left at defaults.  For more information, please read receiver chapter of the <strong><a href=\"https://github.com/betaflight/betaflight/blob/master/docs/Rx.md\" target=\"_blank\" rel=\"noopener noreferrer\">documentation</a></strong>.<br /><span class=\"message-negative\">IMPORTANT:</span>: Do not forget to configure and check failsafe in the Failsafe tab! OpenTx users MUST ensure that the ADC filter is un-checked (disabled) in the Hardware tab of their transmitter! Read the <strong><a href=\"https://github.com/betaflight/betaflight/wiki/Feed-Forward-2.0/_edit#disable-adc-filter\" target=\"_blank\" rel=\"noopener noreferrer\">wiki</a></strong> for instructions."
```

Comment:
Wrong Wiki "Feedforward 2.0" URL, should be https://github.com/betaflight/betaflight/wiki/Feed-Forward-2.0#disable-adc-filter

Reply:
There are actually 2 links in the message. It's there.

Reply:
yes but the current URL is in EDIT mode ;)
Just remove _edit atribute.

Reply:
FIxed!